roles/foreman_provisioning_infrastructure/tasks/main.yml

```diff
@@ -0,0 +1,120 @@
+---
+- name: 'restart foreman-proxy'
```

Comment:
Starting with a restart proxy and foreman doesn't make sense to me. Since foreman-proxy doesn't need the libvirt gems I think this can safely be removed.